replica/replica.go

```diff
@@ -196,23 +215,9 @@ func (replica *Replica) HandleMessage(m Message) {
 			replica.options.Logger.Debugf("ignore message: resync timestamp=%v compared to now=%v", timestamp, now)
 			return
 		}
+		replica.p.HandleMessage(m.Message)
```

Comment:
```suggestion
		replica.p.HandleMessage(m.Message)
		return
```

Reply:
This is needed to make sure that `Resync` is not handled twice, and that whenever it is handled, the delta time check happens.